DemoServer/AdditionalLanguages/python/staticIndexes/additionalSourcesIndex/metadata.json

```diff
@@ -0,0 +1,161 @@
+{
+    "Slug": "additional-sources-index",
+    "SourceFileName": "additionalSourcesIndex.py",
+    "Title": "Additional Sources Index",
+    "DescriptionLines": [
+        "Using the **Additional Sources** feature allows the index's Map & Reduce methods to reference and use code from external source files.",
+        "",
+        "With that, the index definition can be expanded to perform any complex indexing work required."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Additional sources - Blog",
+            "Type" : "link",
+            "Url"  : "https://ayende.com/blog/192001-B/using-machine-learning-with-ravendb"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define the Map index class",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Create the Map index class.",
+                "",
+                "* Use index class type `AbstractIndexCreationTask`.",
```

Comment:
"The class defining index to be created, should inherit AbstractIndexCreationTask" maybe? just a suggestion to think about

Reply:
right - 
C# & Java demos have text with:
`Create the Map index class - inherit from AbstractIndexCreationTask`
=> I will do the same for Python in all places

Reply:
done